### ISSUE TITLE
[remove]

### ISSUE THREAD

**Issue by LgnGithub** [arenatemp/stable-diffusion-webui-model-toolkit#45]
g442h54h

**Comment by arenasys** [comment]
![toolkit_w](https://github.com/arenasys/stable-diffusion-webui-model-toolkit/assets/79061310/0a089b3d-bcc9-48e6-bb10-11f5ace338a3)

**Comment by LgnGithub** [comment]
> ![toolkit_w](https://private-user-images.githubusercontent.com/79061310/316354621-0a089b3d-bcc9-48e6-bb10-11f5ace338a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNjE1MjAsIm5iZiI6MTcxMTM2MTIyMCwicGF0aCI6Ii83OTA2MTMxMC8zMTYzNTQ2MjEtMGEwODliM2QtYmNjOS00OGU2LWJiMTAtMTFmNWFjZTMzOGEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDEwMDcwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkZDVlY2EyODhmMWYwMzljNjgzMTQxMDdkYWFmZDlmMzNiMDE0NjU2ZmFiNzJmM2VmYWZkNzg3OGVhY2Q1YjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gkAk12IPJis10bZjNtLpGxYpCZxDU73lB1aqjmy4IEU)

No, it doesn't exist. Show the context, where can it be found? I enabled your extension and found it nowhere.

**Comment by Asherathe** [comment]
I see it on my screen, just like in the screenshot, right next to the Basic tab. Default settings after using A1111's autoinstall for the extension.

**Comment by cse84** [comment]
![Screenshot_20241130_030818](https://github.com/user-attachments/assets/98440a37-1253-4eba-8c6b-38e248f7e0f5)
The Tab is actually called "Toolkit", not "Advanced". The description should be updated to include the full screenshot, not a screenshot with the important information cropped out. I also had to search for a while to find the right place.

**Comment by Asherathe** [comment]
The Advanced tab referred to is the one you can see in the middle of your screenshot.
